src/polyswarmd/__init__.py

```diff
@@ -66,6 +66,22 @@ def get_auth(api_key, auth_uri):
     return future.result()
 
 
+@cache.memoize(30)
```

Comment:
I was thinking that this was needed last night. PSD needs serious look at performance. We don't have any scans going through stage and there's 19 kappa and 12 zeta instances of PSD running currently. With no scans or load this should be much much lower.